xcp_d/workflows/bold/connectivity.py

```diff
@@ -355,7 +355,6 @@ def init_functional_connectivity_cifti_wf(mem_gb, exact_scans, name='connectivit
                 vmax=1,
             ),
             name='plot_coverage',
-            mem_gb=mem_gb['resampled'],
```

Comment:
Why drop these?

Reply:
We can leave this mystery for another day